pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -915,6 +925,28 @@ const (
 	LegacyNameValidationScheme NameValidationSchemeOptions = "Legacy"
 )
 
+// Specifies the character escaping scheme that will be requested when scraping
+// for metric and label names that do not conform to the legacy Prometheus
+// character set.
+//
+// Supported values are:
+//   - `AllowUTF8NameEscapingScheme` for Full UTF-8 support, no escaping needed.
+//   - `UnderscoresNameEscapingScheme` for Escape all legacy-invalid characters to underscores.
+//   - `DotsNameEscapingScheme` for Escapes dots to `_dot_`, underscores to `__`, and all other
+//     legacy-invalid characters to underscores.
+//   - `ValuesNameEscapingScheme` for Prepend the name with `U__` and replace all invalid
+//     characters with their unicode value, surrounded by underscores.
+//
+// +kubebuilder:validation:Enum=Allow-UTF-8;Underscores;Dots;Values
+type NameEscapingSchemeOptions string
+
+const (
+	AllowUTF8NameEscapingScheme   NameEscapingSchemeOptions = "Allow-UTF-8"
```

Comment:
(nità) https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#constants
```suggestion
	AllowUTF8NameEscapingScheme   NameEscapingSchemeOptions = "AllowUTF8"
```

Reply:
Thanks @simonpasquier 
I got it
Fixed